components/02-elements/swatch/_swatch.scss

```diff
@@ -0,0 +1,79 @@
+$swatch-font-size          : $font-size-small !default;
+$swatch-margin             : 15px !default;
+$swatch-width              : 48px !default;
+$swatch-transition         : $transition-base !default;
+$swatch-background         : $white !default;
+$swatch-label-color        : $gray-light !default;
+$swatch-border-color       : $gray-light !default;
+$swatch-outline-color      : $gray-light !default;
+$swatch-color-hover        : $color-primary !default;
+$swatch-background-active  : $color-primary !default;
+$swatch-border-color-active: $color-primary !default;
+$swatch-color-active       : $white !default;
+$swatch-border             : 3px solid $swatch-border-color !default;
+$swatch-outline            : 3px solid $swatch-outline-color !default;
```

Comment:
I asked to make something with this variables and nothing changed, still, the order is completely random.

Reply:
If you don't know how to do it, just keep alphabetical order.